.setup/bin/recreate_submissions.py

```diff
@@ -288,7 +289,8 @@ def update_egver(g_id: str, user_id: str, active_version: int):
     os.system(f'{regrade_py} {gradeable_path}')
 
 if __name__ == '__main__':
-    if getpass.getuser() != 'root':
+    if os.getuid() == 0:
+        main()
+    else:
         print('Please run this as root, exiting...')
         quit()
```

Comment:
According to the [docs](https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module), `quit()` should not be used outside the interpreter.  Better discussion [here](https://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used).

Why do you need an exit command at all?  Why not just let the program end?

Reply:
You're right, it is unnecessary.